src/Jank/TermManagement/TermList.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Term from './Term';
+import useApi from '../../hooks/useApi';
+import { AddButton, AddLabel, TermInput } from './ui';
+
+type Terms = {
+  _id: string,
+  value: string
+}
+
+const TermList = () => {
+  const [error, terms] = useApi<Terms[]>('/jank/terms');
+
+  if (error) {
+    return <span>Begriffe konnten leider nicht geladen werden</span>;
+  }
+  return (
+    <>
+      <h1>Begriffe verwalten</h1>
+      <form>
+        <AddLabel>
+          Begriff hinzufügen:
+          <TermInput type="text" name="word" />
+        </AddLabel>
+        <AddButton name="Submit">+</AddButton>
+      </form>
+      <ul>
+        {terms && terms.map(({ value }) => (
+          <Term value={value} />
```

Comment:
Whenever you render components in a loop or in an array, each component needs to have its own unique `key` prop. This is so React knows if an array element changes and which element in an array it needs to update. It can't use the array index, because when an element gets deleted or added, in between, the indices will change, yielding unexpected results. As we have a unique `_id` for every `term`, we could just have it like
```suggestion
          <Term key={_id} value={value} />
```

Note that `key` does not need to be defined as a prop on `Term`, this is just a built-in prop that is consumed by React and is not even available in the `Term` component itself.

Reply:
Wanted to add a screenshot but forgot - this is what you see in the browser console when opening the terms page:
![image](https://user-images.githubusercontent.com/481294/83330448-3adff400-a28f-11ea-81eb-5f98f4252558.png)